kolena/_experimental/object_detection/utils.py

```diff
@@ -108,14 +108,22 @@ def _compute_threshold_curve(
         return None
 
     if curve_type == "f1":
-        return Curve(x=thresholds, y=f1s, label=curve_label) if len(f1s) >= 2 else None
-    else:
-        # add a point to start the PR curve on the vertical axis if needed
-        if 0.0 not in recalls:
-            minpos = recalls.index(min(recalls))
-            precisions.append(precisions[minpos])
-            recalls.append(0.0)
-        return Curve(x=recalls, y=precisions, label=curve_label) if len(recalls) >= 2 else None
+        return (
+            Curve(x=thresholds, y=f1s, label=curve_label, extra=dict(Precision=precisions, Recall=recalls))
+            if len(f1s) >= 2
+            else None
+        )
+
+    # add a point to start the PR curve on the vertical axis if needed
+    if 0.0 not in recalls:
+        minpos = recalls.index(min(recalls))
```

Comment:
might need to change f1s and thresholds if there is a length complaint

Reply:
Do you mind taking a look at the unit test failures? Feel free to push to this branch directly.